packages/calcite-components/package.json

```diff
@@ -26,21 +26,21 @@
     "build-storybook": "npm run util:build-docs && NODE_OPTIONS=--openssl-legacy-provider build-storybook --output-dir ./docs --quiet",
     "clean": "npm run util:clean-js-files && npm run util:clean-readmes && rimraf node_modules dist www hydrate docs .turbo",
     "deps:update": "updtr --exclude chalk cheerio typescript @types/jest jest jest-cli ts-jest @whitespace/storybook-addon-html && npm audit fix",
-    "docs": "build-storybook",
-    "docs:preview": "npm run util:build-docs && NODE_OPTIONS=--openssl-legacy-provider STORYBOOK_SCREENSHOT_LOCAL_BUILD=true start-storybook",
+    "screenshot-test": "build-storybook",
+    "screenshot-test-preview": "npm run util:build-docs && NODE_OPTIONS=--openssl-legacy-provider STORYBOOK_SCREENSHOT_LOCAL_BUILD=true start-storybook",
```

Comment:
Make sure to test changes to scripts. This is still referencing `util:build-docs` and is causing an [issue with Chromatic builds](https://github.com/Esri/calcite-design-system/actions/runs/8290501504/job/22688636213?pr=8944#step:6:1):

```
Chromatic CLI v***.0.8
https://www.chromatic.com/docs/cli

Authenticating with Chromatic
    → Connecting to https://index.chromatic.com
Authenticated with Chromatic
    → Using project token '********8823'
Retrieving git information
Retrieved git information
    → Commit '4e8433***' on branch 'brandentheintern/89***-remove-docs-tab'; found *** parent build
Collecting Storybook metadata
No viewlayer package listed in dependencies. Checking transitive dependencies.
Collected Storybook metadata
    → ; no supported addons found
Initializing build
Initialized build
    → Build 4093 initialized
Building your Storybook
    → Running command: npm run build-storybook -- --output-dir /tmp/chromatic--2***42-dkKTNMFhIvej
    → [*                   ]
The CLI tried to run your build-storybook script, but the command failed. This indicates a problem with your Storybook. Here's what to do:

- Check the Storybook build log printed below.
- Run npm run build-storybook or yarn build-storybook yourself and make sure it outputs a valid Storybook by opening the generated index.html in your browser.
- Review the build-storybook CLI options at https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook

Command failed with exit code ***: npm run build-storybook -- --output-dir /tmp/chromatic--2***42-dkKTNMFhIvej
npm ERR! Lifecycle script `util:build-docs` failed with error: 
npm ERR! Error: Missing script: "util:build-docs"

To see a list of scripts, run:
  npm run 
npm ERR!   in workspace: @esri/calcite-components@2.7.0-next.4 
npm ERR!   at location: /home/runner/work/calcite-design-system/calcite-design-system/packages/calcite-components 
npm ERR! Lifecycle script `build-storybook` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @esri/calcite-components@2.7.0-next.4 
npm ERR!   at location: /home/runner/work/calcite-design-system/calcite-design-system/packages/calcite-components 

ℹ Build command:
npm run build-storybook -- --output-dir /tmp/chromatic--2***42-dkKTNMFhIvej
```